package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "budget-app",
+  "version": "1.0.0",
+  "description": "budget app",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/anastasiya-nd/budget-app.git"
+  },
+  "author": "dankoanastasia0@gmail.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/anastasiya-nd/budget-app/issues"
+  },
+  "homepage": "https://github.com/anastasiya-nd/budget-app#readme",
+  "dependencies": {
+    "editorconfig": "^0.15.3"
```

Comment:
You don't need `editorconfig` as a package. You need to add `.editorconfig` file and add an editorconfig plugin to your vscode. Thats it.

Reply:
So, please, remove this package. Use `npm remove editorconfig` command for that.